core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -669,6 +685,34 @@ private[spark] class MapOutputTrackerMaster(
     None
   }
 
+  /**
+   * Return the locations where the Mapper(s) ran. The locations each includes both a host and an
+   * executor id on that host.
+   *
+   * @param dep shuffle dependency object
+   * @param startMapId the start map id
+   * @param endMapId the end map id
```

Comment:
We shouldn't add code that only for future use. For now we only need a single `mapId`, IIUC.